visallo/visallo
# Refactor model-vertexium tests

- [x] mwizeman joeybrk372 rygim jharwig EvanOxfeld

Created a new module named `model-vertexium-test` and moved all `model-vertexium` tests into it. Also updated the tests to make use of helpers and base classes in `core-test`.

The goal of this PR is:
1.) Consolidate all of the test for the `model-vertexium` module into a single place. Currently, the tests covering `model-vertexium` classes are spread out across the `model-vertexium`, `model-vertexium-inmemory`, and `core-test` modules.
2.) Update the `model-vertexium` tests to use the helpers and base classes in `core-test`. Since `core-test` depends on `model-vertexium`, the only way to do this is to create a new module named `model-vertexium-tests`.

Testing Instructions:
Run `mvn clean test` and ensure that all tests pass.

CHANGELOG
Changed: Moved all `model-vertexium` related test classes to a new module named `model-vertexium-test`
